components/Card.js

```diff
@@ -1,49 +1,35 @@
 import Image from "next/image";
 import styled from "styled-components";
-import useLocalStorage from "use-local-storage";
+
 import HeartButton from "./HeartButton";
 import Link from "next/link";
 
-export default function Card({ plant }) {
-  const [ownedPlantIds, setOwnedPlantIds] = useLocalStorage(
-    "ownedPlantIds",
-    []
-  );
-  const isOwned = ownedPlantIds.includes(plant._id);
-
-  function handleToggleOwned() {
-    if (isOwned) {
-      setOwnedPlantIds(ownedPlantIds.filter((id) => id !== plant._id));
-    } else {
-      setOwnedPlantIds([...ownedPlantIds, plant._id]);
-    }
-  }
-
+export default function Card({ plant, handleToggleOwned, isOwned }) {
```

Comment:
a standard naming convention is to call this onToggleOwned, since when you use the Card component, it looks like the other event handler props onClick, etc.